### PR TITLE
invalidate voltage init status on param changes

### PR DESCRIPTION
**Pull request by AbdelHedhili**

### CODE REVIEW

File: src/test/java/org/gridsuite/study/server/VoltageInitTest.java [hunk 1]
```diff
@@ -397,6 +413,18 @@ public void testVoltageInit() throws Exception {
...
+        mockMvc.perform(
+                post("/v1/studies/{studyUuid}/voltage-init/parameters", studyNameUserIdUuid)
+                        .header("userId", "userId")
+                        .contentType(MediaType.ALL)
+                        .content(VOLTAGE_INIT_PARAMETERS_JSON)).andExpect(
+                status().isOk());
+
+        assertTrue(TestUtils.getRequestsDone(2, server).stream().allMatch(r -> r.matches("/v1/parameters/" + VOLTAGE_INIT_PARAMETERS_UUID) || r.matches("/v1/results/invalidate-status.*")));
+        //remove notif about study updating due to parameters changes
+        output.receive(1000);
```
Comment (SlimaneAmar) [inline src/test/java/org/gridsuite/study/server/VoltageInitTest.java:425]: Perhaps just test the notification type for no regression ?

Comment (AbdelHedhili) [reply src/test/java/org/gridsuite/study/server/VoltageInitTest.java:425]: I already did the test above in `testVoltageInitParameters()`

Comment (SlimaneAmar) [reply src/test/java/org/gridsuite/study/server/VoltageInitTest.java:425]: This is what we agreed with the team ?

Comment (SlimaneAmar) [reply src/test/java/org/gridsuite/study/server/VoltageInitTest.java:425]: No purge

Comment (AbdelHedhili) [reply src/test/java/org/gridsuite/study/server/VoltageInitTest.java:415]: changed it

**Review by SlimaneAmar — APPROVED** [review]

**Review by YenguiSeddik — APPROVED** [review]

**Pull request merged**